package.json

```diff
@@ -37,6 +37,7 @@
     "stylelint": "14.16.1",
     "typescript": "^5.2.2",
     "vite": "^5.1.6",
+    "vite-plugin-svgr": "^4.2.0",
```

Comment:
Could you please share an example of how to connect SVG files without using the `vite-plugin-svgr` plugin so that it's possible to interact with the SVG by adding the title attribute?

Reply:
The only approach I currently see is to create a separate component for each SVG file and add the SVG file's content to this component. In this component, we can add the ability to include additional arguments to the SVG wrapper. Is that what you had in mind?